### PR TITLE
Support Linux and add functional tests for vscode client

### PR DESCRIPTION
**Pull request by sowu880**
Support Linux and add functional tests for vscode client

### CODE REVIEW

File: src/Microsoft.Health.Fhir.TemplateManagement.FunctionalTests/OCIArtifactFunctionalTests.cs [hunk 1]
```diff
@@ -42,67 +43,68 @@ public class OCIArtifactFunctionalTests
...
             _testMultiLayersWithValidSequenceNumberImageReference = _containerRegistryServer + "/templatetest:multilayers_valid_sequence";
             _testMultiLayersWithInValidSequenceNumberImageReference = _containerRegistryServer + "/templatetest:multilayers_invalid_sequence";
             _testInvalidCompressedImageReference = _containerRegistryServer + "/templatetest:invalid_image";
-            PushOneLayerWithValidSequenceNumber();
-            PushOneLayerWithoutSequenceNumber();
-            PushOneLayerWithInvalidSequenceNumber();
-            PushMultiLayersWithValidSequenceNumber();
-            PushMultiLayersWithInValidSequenceNumber();
-            PushInvalidCompressedImage();
+            Task.Run(PushOneLayerWithValidSequenceNumberAsync).Wait();
```
Comment (moria97) [inline src/Microsoft.Health.Fhir.TemplateManagement.FunctionalTests/OCIArtifactFunctionalTests.cs:57]: better to use ```IAsyncLifetime``` to execute initialization steps.

**Comment by moria97** [comment]
Do we have to embed two oras-linux binaries?

File: src/Microsoft.Health.Fhir.TemplateManagement.FunctionalTests/OCIArtifactFunctionalTests.cs (hunk 1 above)
Comment (sowu880) [reply src/Microsoft.Health.Fhir.TemplateManagement.FunctionalTests/OCIArtifactFunctionalTests.cs]: Fixed it.

**Comment by sowu880** [comment]
> Do we have to embed two oras-linux binaries?

Fixed it, thanks.

**Review by moria97 — APPROVED** [review]

**Pull request merged**